examples/basic/provider.tf

```diff
@@ -0,0 +1,3 @@
+provider "aws" {
+  region = "eu-central-1"
+}
```

Comment:
I like to see a newline at the end of files but I'm not blocking on that minutia

Reply:
Thanks for the quick replay, tf fmt did this maybe, using it with vscode, but seems not working properly.